.git-blame-ignore-revs

```diff
@@ -0,0 +1,5 @@
+# Migrate code style to Black
```

Comment:
I *think* I understand what this file is doing... but just to be sure, can you clarify for me?

Reply:
`git blame --ignore-revs-file .git-blame-ignore-revs` can be used to ignore large changes like our conversion to black and docformatter to prevent "ruining" the git history. It also can be set a global option in git itself with `git config --global blame.ignoreRevsFile .git-blame-ignore-revs`.